src/components/application_manager/include/application_manager/commands/command_request_impl.h

```diff
@@ -212,6 +212,14 @@ class CommandRequestImpl : public CommandImpl,
   bool HasDisallowedParams() const;
 
   /**
+   * @brief Checks if file name contains invalid symbols e.g. '/'
```

Comment:
@AKalinich-Luxoft Pay attemtion to `IsFileNamePortable(..)` function from Profile. Probably you can move it out to utils and make it available for all needed components. Or write similar function .